brainrender/actors/neurons.py

```diff
@@ -0,0 +1,93 @@
+from pathlib import Path
+from morphapi.morphology.morphology import Neuron as MorphoNeuron
+from vedo import Mesh
+from pyinspect.utils import _class_name
+
+from ..actor import Actor
+
+
+def make_neurons(
+    *neurons, alpha=1, color=None, neurite_radius=8, soma_radius=15, name=None
+):
+    """
+        Returns a list of Neurons given a variable numnber of inputs
+        :param neurons: any accepted data input for Neuron
+        :param alpha: float
+        :param color: str
+        :param neuron_radius: float, radius of axon/dendrites
+        :param soma_radius: float, radius of soma
+        :param name: str, actor name
+    """
+    return [
+        Neuron(
+            n,
+            alpha=alpha,
+            color=color,
+            neurite_radius=neurite_radius,
+            soma_radius=soma_radius,
+            name=name,
+        )
+        for n in neurons
+    ]
+
+
+class Neuron(Actor):
+    def __init__(
+        self,
+        neuron,
+        color=None,
+        alpha=1,
+        neurite_radius=8,
+        soma_radius=15,
+        name=None,
+    ):
+        """
+            Creates an Actor representing a single neuron's morphology
+            :param neuron: path to .swc file, Mesh, Actor or Neuron from morphapi.morphology
+            :param alpha: float
+            :param color: str,
```

Comment:
as suggested above, I  think that support for RGB is convenient as it is what most users would use by default to customise stuff